react/src/components/atoms/forms/Form/Form.stories.js

```diff
@@ -0,0 +1,147 @@
+import React from 'react';
+
+import { storiesOf } from '@storybook/react';
+import { withInfo } from '@storybook/addon-info';
+import {object, withKnobs, array, text} from '@storybook/addon-knobs/react';
+
+import Form, { FormProvider } from './index';
+import FormInfo from './Form.md';
+import InputNumber from '../InputNumber';
+import InputNumberOptions from '../InputNumber/InputNumber.knobs.options';
+import InputSliderOptions from '../InputSlider/InputSlider.knobs.options';
+import InputSlider from '../InputSlider';
+import InputCurrency from '../InputCurrency';
+import InputCurrencyOptions from '../InputCurrency/InputCurrency.knobs.options';
+
+storiesOf('atoms/forms', module).addDecorator(withKnobs)
+  .add('Form', withInfo(`<div>${FormInfo}</div>`)(() => {
+    delete InputNumberOptions.defaultValue;
+    const inputTextOptionsWithKnobs = Object.assign(...Object.entries(InputNumberOptions).map(([k, v]) => (
+      { [k]: v() })));
+    delete InputSliderOptions.ticks;
+    delete InputSliderOptions.domain;
+    delete InputSliderOptions.labelText;
+    const inputSliderOptionsWithKnobs = Object.assign(...Object.entries(InputSliderOptions).map(([k, v]) => (
+      { [k]: v() })));
+    inputSliderOptionsWithKnobs.labelText = text('InputSlider.labelText', 'Slider (Linked to Input 3)');
+    inputSliderOptionsWithKnobs.ticks = object('InputSlider.ticks', { 0: '0%', 60: 'Minimum requirement', 100: '100%' });
```

Comment:
```suggestion
```

This seems like a duplicate for Form.ticks:

![screen shot 2019-02-07 at 9 51 30 am](https://user-images.githubusercontent.com/5789411/52419555-59c8de80-2abe-11e9-81e7-72138f1669a4.png)